core/src/main/scala/cats/data/package.scala

```diff
@@ -3,6 +3,7 @@ package cats
 package object data {
   type NonEmptyStream[A] = OneAnd[Stream, A]
   type ValidatedNel[+E, +A] = Validated[NonEmptyList[E], A]
+  type IorNel[+B, +A] = Ior[NonEmptyList[B], NonEmptyList[A]]
```

Comment:
Personally I cannot see a use-case for `IorNel` as it was before; did you have one in mind? That might help to decide.